client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -312,6 +312,30 @@ export default {
 		);
 	},
 
+	domainManagementTransferToOtherSite( pageContext ) {
+		const state = pageContext.store.getState();
+		const siteId = getSelectedSiteId( state );
+		const isAutomatedTransfer = isSiteAutomatedTransfer( state, siteId );
+		if ( isAutomatedTransfer ) {
+			const siteSlug = getSelectedSiteSlug( state );
+			page.redirect( `/domains/manage/${ siteSlug }` );
+			return;
+		}
+
+		setTitle(
+			i18n.translate( 'Transfer Domain' ),
+			pageContext
+		);
```

Comment:
I like the idea, but I'd rather keep it consistent with the rest of the file for now. Refactoring `Header` in a separate PR (and addressing all the cases) seems like the best way to go 👍 I've created an issue to remember about this: https://github.com/Automattic/wp-calypso/issues/14295